ext/readline/readline.stub.php

```diff
@@ -6,7 +6,7 @@ function readline(?string $prompt = null): string|false {}
 
 /**
  * @param int|string|bool|null $value
- * @return array|int|string|bool|null
+ * @return array<string, int|string|bool>|int|string|bool|null
```

Comment:
Might make sense to just keep this at `mixed` (looks like the docs already do)...

Reply:
Though I don't mind keeping it as you have it.

Reply:
I made it `mixed` at last.